### ISSUE TITLE
None of the examples using PMREMGenerator.fromScene work on iPhone while it works on iPad.

### ISSUE THREAD

**Issue by patrickmatte** [mrdoob/three.js#25888]
### Description

None of the three examples using PMREMGenerator.fromScene render correctly on the iPhone. The envMap is just black. It works correctly on the iPad. Is this a known issue? Does anyone know if there's a fix for it? Or is this just a bug with Safari on iPhone.

https://threejs.org/examples/?q=misc#misc_exporter_usdz
https://threejs.org/examples/#webgl_animation_keyframes
https://threejs.org/examples/?q=gltf#webgl_loader_gltf_compressed
https://threejs.org/examples/?q=gltf#webgl_loader_gltf_sheen
https://threejs.org/examples/?q=ldraw#webgl_loader_ldraw
https://threejs.org/examples/?q=lottie#webgl_loader_texture_lottie
https://threejs.org/examples/?q=hdr#webgl_materials_envmaps_hdr
https://threejs.org/examples/?q=mor#webgl_morphtargets_face
https://threejs.org/examples/?q=nodes#webgl_nodes_loader_gltf_sheen
https://threejs.org/examples/?q=shaders#webgl_shaders_ocean

### Reproduction steps

I can reproduce this bug with every three example using PMREMGenerator.fromScene on the iPhone.

### Code

```js
// code goes here
```

### Live example

https://threejs.org/examples/?q=misc#misc_exporter_usdz
https://threejs.org/examples/#webgl_animation_keyframes
https://threejs.org/examples/?q=gltf#webgl_loader_gltf_compressed
https://threejs.org/examples/?q=gltf#webgl_loader_gltf_sheen
https://threejs.org/examples/?q=ldraw#webgl_loader_ldraw
https://threejs.org/examples/?q=lottie#webgl_loader_texture_lottie
https://threejs.org/examples/?q=hdr#webgl_materials_envmaps_hdr
https://threejs.org/examples/?q=mor#webgl_morphtargets_face
https://threejs.org/examples/?q=nodes#webgl_nodes_loader_gltf_sheen
https://threejs.org/examples/?q=shaders#webgl_shaders_ocean

### Screenshots

![IMG_1983](https://user-images.githubusercontent.com/435963/233195885-a8ca3ab4-4f25-4239-9386-0f20602ee9f2.PNG)
![IMG_1982](https://user-images.githubusercontent.com/435963/233195888-39a79dc7-1b3f-40c0-be44-6498a6d66d46.PNG)
![IMG_1981](https://user-images.githubusercontent.com/435963/233195890-fc064bbd-3200-4c2e-901d-7c603769c1a2.PNG)
![IMG_1980](https://user-images.githubusercontent.com/435963/233195893-a204b55f-de6a-4667-bd49-d47270a57b19.PNG)
![IMG_1979](https://user-images.githubusercontent.com/435963/233195895-bdf983f5-1c55-4995-93a1-6f6c07dbfbc2.PNG)
![IMG_1978](https://user-images.githubusercontent.com/435963/233195902-f034507d-9197-4359-b914-2b6eeff1bf8c.PNG)
![IMG_1977](https://user-images.githubusercontent.com/435963/233195906-a1bfe3a6-24ba-44db-b01f-aae2d7a21388.PNG)
![IMG_1976](https://user-images.githubusercontent.com/435963/233195911-f7d70ddd-6d55-4921-be07-d25842d610f1.PNG)
![IMG_1975](https://user-images.githubusercontent.com/435963/233195915-5e7ec69a-bfd5-4040-b5c2-39e5c12f7d3f.PNG)

### Version

r151

### Device

Mobile

### Browser

Safari

### OS

iOS

**Comment by donmccurdy** [comment]
Testing these examples on iPhone 11 Pro, iOS 16.4.1, I see the same problem.

**Comment by Mugen87** [comment]
We have lately received more than one report about issues on iOS 16.4 due to WebGL regressions in Webkit, e.g. #25862 or #25736. Since there are not errors or warnings in the console and no obvious workarounds (like disabling anti-aliasing or using WebGL 1), I believe this needs to be fixed on the browser side.

**Comment by hurshhazari** [comment]
I think some of the changes post iOS 16.4 have been related to WebGL and metal. Some experimentation that worked for our team was by manually disabling `WebGL via Metal` (this is enabled by default).

Try updating this property via:

Settings -> Safari -> Advanced -> Experimental Features -> WebGL via Metal

![webgl](https://user-images.githubusercontent.com/29025711/233379403-f9b1b3bf-3049-4b5a-9428-294f4c7720e7.png)

**Comment by donmccurdy** [comment]
True, but I think it's helpful to share these workarounds in case it unblocks someone else. They may be trying to develop an app that hasn't been launched yet, for example.

It appears the WebKit team is fixing the issues, though I am not sure when those fixes will be released:

- https://bugs.webkit.org/show_bug.cgi?id=254739
- https://bugs.webkit.org/show_bug.cgi?id=252237
- https://bugs.webkit.org/show_bug.cgi?id=252585

**Comment by Mugen87** [comment]
@patrickmatte Can you please verify if iOS 16.5 fixes the issue?
